From https://ftp.gnu.org/gnu/bash/bash-5.0-patches/bash50-006

Signed-off-by: Pascal de Bruijn <p.debruijn@unilogic.nl>

			     BASH PATCH REPORT
			     =================

Bash-Release:	5.0
Patch-ID:	bash50-006

Bug-Reported-by:	Tomas Mozes <hydrapolic@gmail.com>
Bug-Reference-ID:	<CAG6MAzQumLU2vhnmr1UrYAUQAFW5Yo8hfM_sEibX9RJQWJkRrA@mail.gmail.com>
Bug-Reference-URL:	http://lists.gnu.org/archive/html/bug-bash/2019-03/msg00037.html

Bug-Description:

Bash-5.0 did not build successfully if SYSLOG_HISTORY was defined without
also defining SYSLOG_SHOPT.

Patch (apply with `patch -p0'):

*** ../bash-5.0-patched/builtins/shopt.def	2018-10-05 14:49:02.000000000 -0400
--- b/builtins/shopt.def	2019-01-23 09:55:22.000000000 -0500
***************
*** 123,127 ****
  #endif
  
! #if defined (SYSLOG_HISTORY) && defined (SYSLOG_SHOPT)
  extern int syslog_history;
  #endif
--- 123,127 ----
  #endif
  
! #if defined (SYSLOG_HISTORY)
  extern int syslog_history;
  #endif
*** ../bash-5.0/patchlevel.h	2016-06-22 14:51:03.000000000 -0400
--- b/patchlevel.h	2016-10-01 11:01:28.000000000 -0400
***************
*** 26,30 ****
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 5
  
  #endif /* _PATCHLEVEL_H_ */
--- 26,30 ----
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 6
  
  #endif /* _PATCHLEVEL_H_ */
